openhands/runtime/utils/request.py

```diff
@@ -29,7 +29,8 @@ def is_retryable_error(exception: Any) -> bool:
     )
 
 
-@retry(
+# Type ignore is needed because tenacity's retry decorator doesn't preserve function signature
+@retry(  # type: ignore
```

Comment:
Hmm doesn't seem like a problem elsewhere

Maybe worth double checking if this is true - not blocking imo

Reply:
Thanks, you were correct here, I was too trusting :)